src/pouchdb/getLinks.ts

```diff
@@ -31,6 +31,11 @@ export const offlineDoctypes = [
   'io.cozy.apps.suggestions',
   'io.cozy.triggers',
   'io.cozy.apps_registry',
+  'io.cozy.calendar.events',
+  'io.cozy.calendar.todos',
+  'io.cozy.calendar.presence',
+  'io.cozy.timeseries.grades',
+  'io.cozy.identities',
```

Comment:
It's ok for now, but eventually, we probably should think to a better way to declare doctypes, it is a bit cumbersome for now